app/assets/sass/patterns/_check-your-answers.scss

```diff
@@ -1,14 +1,84 @@
-.check-your-answers {
 
-  td {
-    @include core-19;
-    vertical-align: top;
+// please adjust, depending on length of questions and answers and number of sections
+$cya-first-column-width: 30% !default;
```

Comment:
I had a smaller comment in the CSS further down. Do you think I should change anything about it?